### PR TITLE
feat: need a review

### PR DESCRIPTION
**Pull request by steviebiddles**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -9,4 +9,10 @@ app.get('/', (req, res) => {
   });
 });
 
+app.get('/review', (req, res) => {
+  res.send({
+    message: 'need a review',
```
Comment (JackOHara) [inline src/app.js:14]: I would suggest putting this message in all caps and adding an exclamation mark

Comment (steviebiddles) [reply src/app.js]: Done.

**Comment by steviebiddles** [comment]
Kudos to JackOHara! You have 500 YEI points coming your way.

**Pull request merged**